src/js/player.js

```diff
@@ -19,7 +22,7 @@ import log, { createLogger } from './utils/log.js';
 import toTitleCase, { titleCaseEquals } from './utils/to-title-case.js';
 import { createTimeRange } from './utils/time-ranges.js';
 import { bufferedPercent } from './utils/buffer.js';
-import * as stylesheet from './utils/stylesheet.js';
+// import * as stylesheet from './utils/stylesheet.js';
```

Comment:
```suggestion

```

Reply:
Yeah, this is one of those things that we tried to remove and it ended up giving us a compiler error, so we left it.